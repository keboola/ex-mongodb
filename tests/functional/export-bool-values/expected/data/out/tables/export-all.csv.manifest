{"incremental":false,"columns":["id","bool"],"column_metadata":{"id":[{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"string"}],"bool":[{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"string"}]}}