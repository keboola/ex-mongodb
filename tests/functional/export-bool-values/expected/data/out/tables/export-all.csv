"id","bool"
"5716054bee6e764c94fa7ddd","1"
"5716054bee6e764c94fa8181","0"
"5716054bee6e764c94fa8213",""
